custom_app/backend.Dockerfile

```diff
@@ -1,8 +1,20 @@
-ARG FRAPPE_VERSION
-FROM frappe/frappe-worker:${FRAPPE_VERSION}
+# syntax=docker/dockerfile:1.3
+
+ARG ERPNEXT_VERSION
+FROM frappe/erpnext-worker:${ERPNEXT_VERSION}
+
+USER root
 
 ARG APP_NAME
-COPY --chown=frappe . ../apps/${APP_NAME}
+COPY . ../apps/${APP_NAME}
+
+RUN --mount=type=cache,target=/root/.cache/pip \
+    install-app ${APP_NAME}
+
+# or with git:
+# ARG APP_NAME
+# ARG BRANCH
+# ARG GIT_URL
+# RUN install-assets ${APP_NAME} ${BRANCH} ${GIT_URL}
 
-RUN echo "frappe\n${APP_NAME}" >/home/frappe/frappe-bench/sites/apps.txt \
-    && ../env/bin/pip install --no-cache-dir -e ../apps/${APP_NAME}
+USER frappe
```

Comment:
Can uid/gid be `--build-arg` defaulting to 1000/frappe?

People who really need different gid/uid can override it in their custom apps then.

https://github.com/frappe/frappe_docker/issues/645

Reply:
It would be quite confusing if users would rebuild images themselves with different uid and name them the same way official images are named. In this case I would suggest to add another Dockerfile and change user there.

Reply:
Part of docs I guess. I'll resolve.